domains/transfer/Transfer.API/API/ApiModels/Requests/RuleExtensions.cs

```diff
@@ -13,4 +14,25 @@ public static class RuleExtensions
         => ruleBuilder
             .LessThan(253402300800)
             .WithMessage("{PropertyName} too high! Please make sure the format is UTC in seconds.");
+
+    public static IRuleBuilderOptions<T, string> MustBeValidBase64EncodedWalletDepositEndpoint<T>(this IRuleBuilder<T, string> ruleBuilder)
+            => ruleBuilder
+                .Must(IsValidBase64String)
+                .WithMessage("Base64-encoded Wallet Deposit Endpoint is not valid");
+
+    private static bool IsValidBase64String(string base64)
+    {
+        if (string.IsNullOrEmpty(base64))
+            return false;
+
+        try
+        {
+            _ = Convert.FromBase64String(base64);
+            return true;
+        }
+        catch (FormatException)
+        {
+            return false;
+        }
```

Comment:
Use `TryFromBase64String` instead to avoid exception control flow and let dotnet decide the best and fast way of determining if it is valid bas64.

Reply:
Fixed in https://github.com/Energinet-DataHub/energy-origin/pull/443/commits/b1e59cfb958323bc2e4feea8250c16f0b542af86